tests/transaction_history_test.py

```diff
@@ -0,0 +1,105 @@
+import unittest
+import unittest.mock as mock
+import json
+import os
+import sys
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+import server
+from tests.profileview_test import mocked_bad_query
+
+KEY_INPUT = "input"
+KEY_EXPECTED = "expected"
+KEY_TICKET_TRANSACTION = "ticketTransaction"
+KEY_ID = "id"
+KEY_DATETIME = "datetime"
+KEY_ACTIVITY = "activity"
+KEY_AMOUNT = "amount"
+
+def mocked_iterable():
+    mocked_iter = mock.Mock()
+    mocked_iter.all.return_value = iter([MockedTransData()])
+    return mocked_iter
+
+def mocked_transaction_query(table):
+    if str(table) == "<class 'server.models.transaction.Transaction'>":
+        mocked_query_all = mock.Mock()
+        mocked_query_all.filter.return_value = mocked_iterable()
+        return mocked_query_all
+    return "The given table was not Transaction."
+
+class MockedTransData():
+    id = 1
+    datetime = "Tue, 10 Nov 2020 18:19:43 GMT"
+    activity = "COINFILP"
+    ticket_amount = 50
+
+class ProfileViewTest(unittest.TestCase):
+    def setUp(self):
+        server.app.config['SECRET_KEY'] = os.getenv("SECRET_KEY")
```

Comment:
Add default value for os.getenv

Reply:
Also, why are you setting the secret key here? Isn't it already set in the server/__init__.py?